dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java

```diff
@@ -68,7 +68,7 @@ public void synchronousWrite(Object event) {
         GenericData genData = null;
         Integer levelVal = null;
         if (event instanceof LogTraceData) {
-            genData = ((LogTraceData) event).getGenData();
+            genData = (LogTraceData) event;
```

Comment:
at start of this method, need to check to make sure event is instanceof GenericData.  If it's not GenericData we should throw an IllegalArgumentException

Reply:
ok

Reply:
Added an else statement below.